src/server/editorServices.ts

```diff
@@ -410,8 +408,8 @@ namespace ts.server {
 
             Debug.assert(!!this.host.createHash, "'ServerHost.createHash' is required for ProjectService");
 
+            this.currentDirectory = this.host.getCurrentDirectory();

```

Comment:
Thats used for absolute path conversions.. It was already passed to createDocumentRegistry in the constructor.. I just stored it here so it doesnt go to host all the time.